frontend/src/config/data-provider-configurations.tsx

```diff
@@ -132,6 +135,7 @@ export const DATA_PROVIDER_CONFIGURATIONS: {
     email: 'support@entreprise.api.gouv.fr',
     type: DataProviderType.api,
     component: ApiEntreprise,
+    url: 'https://entreprise.api.gouv.fr/compte/jetons',
```

Comment:
@skelz0r we want to add direct link to the API manager from our enrollment list. 

Is that normal that for API Particulier the URL is https://particulier.api.gouv.fr/compte and for API Entreprise it has an additional `/jetons` in the path ?

Reply:
It's a legacy url, we migrate it from the legacy with iso features/specs.

Is it a problem for you ? Because I can add an alias between `/compte` and `/compte/jetons`

Reply:
no we are ok with it. It's just to double check with you that you are aware of this difference.